_posts/2019-04-16-sicko-mode.md

```diff
@@ -0,0 +1,100 @@
+miss the song’s true message: the story of a young man exploring his sexuality.
+
+<iframe allow="autoplay *; encrypted-media *;"
+style="width:100%;max-width:660px;overflow:hidden;background:transparent;"
+sandbox="allow-forms allow-popups allow-same-origin allow-scripts
+allow-storage-access-by-user-activation
+allow-top-navigation-by-user-activation"
+src="https://embed.music.apple.com/
+us/album/sicko-mode/1421241217?i=1421242781&app=music"
+height="150" frameborder="0"></iframe>
+
+The first beat of the song is mild and unassuming - a careful allegory chosen
+to represent our society’s heteronormativity. When our protagonist is in the
+stage this song describes (i.e. has still not come out), by obeying the
+heterosexual norms of society, he is repressed just like the beat of the song.
+Aubrey “Drake” Graham (hereby solely referred to as Drake), plays a close
+friend of our protagonist. He notes that as for heterosexuality, “that shit all
+he know, he don’t know nothin’ else”. Unable to see our hero living a lie,
+our artists take action by awakening the protaganist's true sexuality (“I
+tried to show him, yea I tried to show him”). Their attempt is successful, as
+they note our character is “goin’ on you with the pick and roll”. For readers
+unfamiliar with basketball, a pick and roll involves a player forcing a
+defender to *switch* to defending him by setting up a screen. These words are
+accompanied by a beat switch -- and also accompanied by our character’s switch
+of sexuality.
+
+The second beat is more aggressive and powerful, consistent with our
+character’s newfound interest in masculinity.
+
+> Different coloured chains, think my jeweller really sellin’ fruits
+
+ He immediately affirms his new sexuality by buying multicoloured jewellery,
+which is associated with gay pride. Furthermore “fruit” is common slang for
+homosexuals, further alluding to his jeweller selling symbols of homosexuality.
+
+> Most of these girls ain’t got a clue
+
+Not only does our character no longer see women as potential sexual partners,
+he goes so far as to deride their intelligence as a whole.
+
+> We ain’t even make it to room.  
+> She thought it was the ocean, it’s just a pool.
+
+Here, as a woman attempts to engage in promiscuous behaviour, our hero denies
+her access to his room (and therefore himself). Furthermore, he corrects her --
+his desire for women cannot be compared to the vast ocean, but just a small
+pool, by comparison practically non-existent.
+
+However, due to the anti-homosexual sentiments still present in many American
+communities, our protagonist is unable to sustain his true desires. Once they
+find out, he is physically assaulted and knocked “out like a light **(ey-uh)**”
+(emphasis mine). The beat change here again represents another change in his
+life. He must settle for practising his sexuality in secret and pretending to
+obey heterosexual norms.
+
+He regrets this as he notes “this shit got double bedroom”, meaning he
```

Comment:
Put comma inside quotes

Reply:
Again, valid otherwise.

Reply:
It's not valid. See my previous link and screenshot

Reply:
Agree with Ming on this one; since the quotes are used to describe speech, commas go inside the marks.